src/main/kotlin/com/autonomousapps/internal/advice/filter/FilterSpec.kt

```diff
@@ -114,4 +119,17 @@ internal class FilterSpec(
       universalFilter.predicate(dependency)
     }
   }
+
+  val unusedProcsAdviceFilter: (Dependency) -> Boolean = { dependency ->
+    if (anyBehavior is Ignore || unusedProcsBehavior is Ignore) {
+      // If we're ignoring everything or just ignoring all compileOnly dependencies, then do that
+      false
+    } else if (anyBehavior.filter.plus(unusedProcsBehavior.filter).contains(dependency.identifier)) {
+      // If we're ignoring some specific dependencies, then do that
+      false
+    } else {
+      // If the dependency is universally ignored, do that
+      universalFilter.predicate(dependency)
```

Comment:
Latest master has changed the convention somewhat. These two lines are now generally:
```kotlin
// If the dependency is universally ignored, or is a facade we care about, ignore it
universalFilter.predicate(dependency) && facadeFilter.predicate(hasDependency)
```
although it seems highly unlike an annotation processor would trigger the facade filter. :thinking:

Reply:
Let's keep the facade. Who knows ?